lib/model-access-wrapper.js

```diff
@@ -607,7 +607,22 @@ class ModelAccessWrapper extends CrispHooks {
 	_checkDiffPermissions(permissions, doc, data) {
 		const grant = permissions.getTargetGrant(this.options.permissionsTarget, data);
 		const patch = Update.createFromDiff(doc.data, data);
-		const fields = _.pull(createUpdate(patch).getUpdatedFields(), '_id');
```

Comment:
It was pulling `_id` because that was part of one of my old solutions to this, and was only added recently in one of my prior PRs.  The way it works now makes this unnecessary, as `_id` is never present.